modules/oe_content_entity/tests/src/Kernel/CorporateEntityAccessControlHandlerTest.php

```diff
@@ -189,6 +201,16 @@ protected function accessTestScenarios(): array {
         'expected_result' => AccessResult::neutral()->addCacheContexts(['user.permissions'])->addCacheTags(['oe_corporate_entity_test:1']),
         'status' => 1,
       ],
+      'user with create, update, delete access / view label / published entity' => [
+        'permissions' => [
+          'create oe_corporate_entity_test test_bundle corporate entity',
+          'edit oe_corporate_entity_test test_bundle corporate entity',
+          'delete oe_corporate_entity_test test_bundle corporate entity',
+        ],
+        'operation' => 'view label',
+        'expected_result' => AccessResult::neutral()->addCacheContexts(['user.permissions'])->addCacheTags(['oe_corporate_entity_test:1']),
+        'status' => 1,
+      ],
```

Comment:
As I understand these test cases are applied for access `view label` operation of the **corporate entity**, but according to changes it has to be applied to **corporate type entity** (bundle).